README.md

```diff
@@ -31,7 +31,7 @@ az hanainstance update --resource-group $RESOURCE_GROUP --instance-name $HANA_IN
 To update a key-value pair in the Tags field of a specific HANA instance:
 
 ```
-az hanainstance update --resource-group $RESOURCE_GROUP --instance-name $HANA_INSTANCE_NAME --set tags.key=udpatedValue
+az hanainstance update --resource-group $RESOURCE_GROUP --instance-name $HANA_INSTANCE_NAME --set tags.key=updatedValue
```

Comment:
What changed on this line? Is it related to monitoring?

Reply:
not related to monitoring, I found a typo and thought I would fix. I can fix in a different PR, but thought it would be ok to make the change here since it's so small. I just changed "udpatedValue" to "updatedValue"

Reply:
Yeah, I prefer this kind of things to be on a different PR to prevent it being reverted if this commit ever needs to be reverted.

Reply:
ok, will remove from here and send out another PR to fix that